sdk/core/System.ClientModel/src/Pipeline/ClientPipeline.cs

```diff
@@ -102,9 +110,7 @@ public static ClientPipeline Create(ClientPipelineOptions options, params Pipeli
 
         int perTryIndex = index;
 
-        TimeSpan networkTimeout = options.NetworkTimeout ?? PipelineResponse.DefaultNetworkTimeout;
-        ResponseBufferingPolicy bufferingPolicy = new(networkTimeout);
-        policies[index++] = bufferingPolicy;
+        policies[index++] = new ResponseBufferingPolicy();
```

Comment:
Since this ctor no longer takes any arguments, should we create a static instance and use that, like we do with some other policies.